include/cupla/manager/Event.hpp

```diff
@@ -61,7 +61,8 @@ namespace detail
             hasTimer( !( flags & cuplaEventDisableTiming ) ),
             event(
                 new AlpakaEvent(
-                    Device< T_DeviceType >::get().current()
+                    Device< T_DeviceType >::get().current(),
+                    !(flags & cuplaEventBlockingSync)
```

Comment:
please add here `#if( ALPAKA_ACC_GPU_HIP_ENABLED == 1 || ALPAKA_ACC_GPU_CUDA_ENABLED == 1 )` to handle the interface differences in alpaka.

Reply:
Should be fixed now.